tests/post/steps/test_dns.py

```diff
@@ -0,0 +1,66 @@
+from pytest_bdd import scenario, then, parsers
+from contextlib import contextmanager
+from kubernetes import client, config
+import os
+import time
+import yaml
+
+
+@contextmanager
+def run_inside_busybox(tmp_path, host, *args, **kwd):
+    # Copy the admin.conf from the bootstrap
+    kubeconfig_file = str(tmp_path/"admin.conf")
+    with host.sudo():
+        admin_content = host.check_output(
```

Comment:
Is it not easier to write:
```
host.file("/tmp/foo").content_string
```

Reply:
indeed I looked for sth like that but I certainly missed it ! thnx